src/window.rs

```diff
@@ -92,23 +92,19 @@ impl WindowId {
     pub const fn dummy() -> Self {
         WindowId(platform_impl::WindowId::dummy())
     }
-}
 
-impl fmt::Debug for WindowId {
-    fn fmt(&self, fmtr: &mut fmt::Formatter<'_>) -> fmt::Result {
-        self.0.fmt(fmtr)
+    pub const fn into_raw(self) -> u64 {
+        self.0.into_raw()
     }
-}
 
-impl From<WindowId> for u64 {
-    fn from(window_id: WindowId) -> Self {
-        window_id.0.into()
+    pub const fn from_raw(id: u64) -> Self {
+        Self(platform_impl::WindowId::from_raw(id))
     }
 }
 
-impl From<u64> for WindowId {
-    fn from(raw_id: u64) -> Self {
-        Self(raw_id.into())
+impl fmt::Debug for WindowId {
+    fn fmt(&self, fmtr: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.0.fmt(fmtr)
     }
 }
 
```

Comment:
Can't this be derived at this point?

Reply:
This is done on a bunch of times I assume to improve the `Debug` output, I believe it turns `WindowId(WindowId(0))` to `WindowId(0)`.

Reply:
See https://github.com/rust-windowing/winit/pull/3622